snmp/backupninja.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+import io
+import re
+import gzip
+import os
+import json
+
+LOGFILES = [
+    "/var/log/backupninja.log",
+    "/var/log/backupninja.log.1.gz",
+]
+
+# Last status
+last_actions = 0
+last_fatal = 0
+last_error = 0
+last_warning = 0
+
+def main():
+    get_backupninja_state()
+    output = {'actions': last_actions, 'fatal': last_fatal, 'error': last_error, 'warning': last_warning}
+    print(json.dumps(output))
+
+
+def readlog(logfile):
+    if logfile.endswith('.gz'):
+        return gzip.open(logfile,'r')
+    else:
+        return io.open(logfile,'r')
+
+
+def get_backupninja_state():
+    global last_actions, last_fatal, last_error, last_warning
```

Comment:
using global is not such elegant,

better:

```
def get_backupninja_state():
    last_actions = 0
    last_fatal = 0
    last_error = 0
    last_warning = 0
 ... do some stuff ...
    return last_actions, last_fatal, last_error, last_warning
```

and in main:
`last_actions, last_fatal, last_error, last_warning = get_backupninja_state()`